src/Ext/CaptureManager/Body.cpp

```diff
@@ -47,7 +47,7 @@ bool CaptureManagerExt::FreeUnit(CaptureManagerClass* pManager, TechnoClass* pTa
 				auto pOriginOwner = pNode->OriginalOwner->Defeated ?
 					HouseClass::FindNeutral() : pNode->OriginalOwner;
 
-				pTarget->SetOwningHouse(pOriginOwner);
+				pTarget->SetOwningHouse(pOriginOwner, !bSilent);
```

Comment:
```suggestion
				pTarget->SetOwningHouse(pOriginOwner, !silent);
```

There are more uses of `bSilent` above this diff, but I can't mark them in suggestions, so they have to be changed manually.